.github/workflows/macos.yml

```
@@ -36,20 +36,33 @@ jobs:
 
       - name: Install toolchain
         id: toolchain
-        uses: actions-rs/toolchain@master
+        uses: actions-rs/toolchain@v1.0.7
```

```suggestion
        uses: actions-rs/toolchain@v1
```
It is recommended to use this tag as it following updates to the latest minor version